instruments/tests/test_config.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module containing tests for util_fns.py
+"""
+
+# IMPORTS ####################################################################
+
+from __future__ import absolute_import, unicode_literals
+
+from io import StringIO
+
+from unittest import skipIf
+
+import quantities as pq
+
+try:
+    import yaml
+except ImportError:
+    yaml = None
+
+from instruments import Instrument
+from instruments.config import (
+    load_instruments
+)
+
+# TEST CASES #################################################################
+
+# pylint: disable=protected-access,missing-docstring
+
+@skipIf(yaml is None, "PyYAML is not installed.")
```

Comment:
Fair point, I'll make sure YAML (whichever once we go with) is appropriately a dependency for the test environment, then. I think now that the ``conda`` command depends on ``rumel.yaml``, it's not as heavy of a dependency as it used to be.